src/qt/receivecoinsdialog.cpp

```diff
@@ -267,9 +267,18 @@ void ReceiveCoinsDialog::copyColumnToClipboard(int column)
 // context menu
 void ReceiveCoinsDialog::showMenu(const QPoint &point)
 {
-    if (!selectedRow().isValid()) {
+    QModelIndex sel = selectedRow();
+    if (!sel.isValid()) {
         return;
     }
+
+    // disable context menu actions when appropriate
+    const RecentRequestsTableModel * const submodel = model->getRecentRequestsTableModel();
+    const RecentRequestEntry *req = &submodel->entry(sel.row());
```

Comment:
nit: Pointer `req` is not change within function, could be  `const RecentRequestEntry * const req` (same as `submodel` above).

Reply:
addressed in 336a643